### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, lstat '/Users/irichter/Diary'

### ISSUE THREAD

**Issue by ingorichter** [sluedecke/atom-diary#1]
[Enter steps to reproduce below:]
1. Create a new entry (the first one)

**Atom Version**: 1.2.4
**System**: Mac OS X 10.11.1
**Thrown From**: [atom-diary](https://github.com/sluedecke/atom-diary) package, v1.0.1
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/Users/irichter/Diary'

```
At fs.js:844

Error: ENOENT: no such file or directory, lstat '/Users/irichter/Diary'
    at Error (native)
    at fs.lstatSync (fs.js:844:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:189:16)
    at Object.realpathSync (fs.js:1454:21)
    at Object.fs.realpathSync (ATOM_SHELL_ASAR.js:262:29)
    at Object.module.exports.getCreateDirectories.getCreateDirectories [as getCreateDirectories] (/Users/irichter/.atom/packages/atom-diary/lib/calendar-lib.coffee:93:18)
    at Object.module.exports.AtomDiary.addEntry (/Users/irichter/.atom/packages/atom-diary/lib/atom-diary.coffee:158:16)
    at CalendarView.module.exports.CalendarView.addEntry (/Users/irichter/.atom/packages/atom-diary/lib/calendar-view.coffee:241:11)
    at HTMLLabelElement.<anonymous> (/Users/irichter/.atom/packages/atom-diary/lib/calendar-view.coffee:3:1)
```
### Commands

```
     -0:11.7.0 atom-diary:toggleCalendar (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
```
### Config

``` json
{
  "core": {
    "themes": [
      "seti-ui",
      "seti-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
asciidoc-preview, v0.6.0
atom-diary, v1.0.1
language-asciidoc, v0.12.0
linter-coffeescript, v1.0.0
seti-syntax, v0.4.1
seti-ui, v0.8.1
webbox-color, v0.5.9

# Dev
No dev packages
```

**Comment by sluedecke** [comment]
Bug reproduced, exists on Linux as well (probable issue with mkdirp package)

**Comment by sluedecke** [comment]
Fixed in 1.1.0
